polyfills/Array/from/polyfill.js

```diff
@@ -1,36 +1,104 @@
-Object.defineProperty(Array, 'from', {
-	configurable: true,
-	value: function from(source) {
-		// handle non-objects
-		if (source === undefined || source === null) {
-			throw new TypeError(source + ' is not an object');
+(function arrayFromPolyfill() {
```

Comment:
Ah, I'd forgotten about that, thanks. Is that true in all the browsers we support? (IE6+, FF3.6, Saf4, Chrome1)?